package.json

```diff
@@ -2,6 +2,7 @@
   "name": "rdmo",
   "scripts": {
     "build:prod": "webpack --config webpack.config.js --mode production",
+    "build:testing": "webpack --config webpack.config.js --mode production --env testing --fail-on-warnings",
```

Comment:
Why not just --mode testing and add `'process.env.NODE_ENV': JSON.stringify('testing')`?

Reply:
When I read the docs, I don't think it goes like that:
[mode ]( https://webpack.js.org/configuration/mode/) gets only `string = 'production': 'none' | 'development' | 'production'`
for [`--env`](https://webpack.js.org/api/cli/#env) I can pass anything

Reply:
alright, I did not check.